app/assets/templates/directives/actions-menu.pug

```diff
@@ -36,5 +33,5 @@
       menu-row(
         faded='true', 
         label="'No Actions Available'", 
-        ng-if='extension.actionsWithContextForItem(self.props.item).length == 0'
+        ng-if='extension.actionsWithContextForItem(self.item).length === 0'
```

Comment:
Does `===` work in templates? Not actually sure...something to keep in mind if we experience issues here.

Reply:
Yeah, it works on templates. In this case I think we should be fine with `==`, which checks the value is equal to 0.